.taskcluster.yml

```diff
@@ -138,17 +138,61 @@ tasks:
               - "git clone --recursive --quiet ${repository} &&
                  cd rust-code-analysis &&
                  git -c advice.detachedHead=false checkout ${head_rev} &&
-                 cargo build --release &&
+                 cargo build --all --release &&
                  cd target/release &&
-                 tar -zvcf /build.tar.gz --transform 's,^,rust-code-analysis-linux-x86_64/,' rust-code-analysis"
+                 mkdir /build.tar.gz &&
+                 mv rust-code-analysis-cli /build.tar.gz"
             artifacts:
-              public/rust-code-analysis-linux-x86_64.tar.gz:
+              public/rust-code-analysis-cli:
                 expires: {$fromNow: '2 weeks'}
                 path: /build.tar.gz
```

Comment:
And this is the path in the Docker container

Reply:
Is the path where I can find the `rust-code-analysis-cli` binary, right? So is it correct in this case?

Reply:
Yep! Although it's super ugly to call the binary "build.tar.gz" :P

Reply:
You're right, I'm going to change that name :)